connector/connect/common/src/main/protobuf/spark/connect/commands.proto

```diff
@@ -177,3 +179,97 @@ message WriteOperationV2 {
   // (Optional) A condition for overwrite saving mode
   Expression overwrite_condition = 8;
 }
+
+message WriteStreamOperation {
```

Comment:
Can you follow https://github.com/apache/spark/blob/master/connector/connect/docs/adding-proto-messages.md over the proto style?

Reply:
What is missing? '(Optional)' or documentation?

Reply:
Both? I saw many of the proto fields has no comment at all?

Reply:
`base.proto` has so many fields without 'Optional' or comment?

Reply:
Optional is always implied. May be it is required only if we want to emphasis optional part of it.